core/templates/dev/head/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -190,25 +190,52 @@ oppia.directive('questionsList', [
               controller: [
                 '$scope', '$uibModalInstance',
                 function($scope, $uibModalInstance) {
-                  $scope.selectedSkillIds = [];
+                  var fillSkillDifficulties = function(length) {
+                    $scope.selectedSkillDifficulties = [];
+                    for (var i = 0; i < length; i++) {
+                      $scope.selectedSkillDifficulties.push('0.3');
```

Comment:
It would be better to set a constant for 0.3 in case the number for each difficulty level is changed later.

Reply:
Maybe call it DEFAULT_SKILL_DIFFICULTY and add a TODO above it explaining this is temporary (that's correct, right?) and what it'll be eventually replaced with.

Reply:
No this is the default value (so that 'Easy' is selected by default)

Reply:
Set it as a constant